packages/design-system-react/src/components/avatar-icon/AvatarIcon.types.ts

```diff
@@ -0,0 +1,63 @@
+import type { ComponentProps } from 'react';
+
+import { IconName } from '../icon';
+import type { IconProps } from '../icon';
+
+export enum AvatarIconSeverity {
+  Default = 'default',
+  Info = 'info',
+  Success = 'success',
+  Error = 'error',
+  Warning = 'warning',
+}
+
+export enum AvatarIconSize {
+  /**
+   * Extra small size (16px)
+   */
+  Xs = 'xs',
+  /**
+   * Small size (24px)
+   */
+  Sm = 'sm',
+  /**
+   * Medium size (32px)
+   */
+  Md = 'md',
+  /**
+   * Large size (40px)
+   */
+  Lg = 'lg',
+  /**
+   * Extra large size (48px)
+   */
+  Xl = 'xl',
+}
```

Comment:
Previously, I exported `AvatarBaseSize as AvatarIconSize` from the `AvatarIcon` index, but it caused circular dependencies. After some consideration, I think this is bad practice and could lead to issues, so we'll need to revert to defining individual enums.